### PR TITLE
Article style consistency edits

### PR DESCRIPTION
**Pull request by itsalyse**
Updates existing support articles to have the correct capitalization for Integrated Providers, Integrated DNS Providers, and Integrated Domain Providers. 

Updates style to capitalize H1 & lowercase H2 as per https://support.dnsimple.com/articles/writing-guide/ (will be conducting an audit for this and other issues across all support articles in the near future)

### CODE REVIEW

**Comment by itsalyse** [comment]
@SarahCaminiti1 @laetitiaeden these should be fine (there were no copy updates, just capitalization/punctuation) but need another set of eyes on them before I merge

**Review by SarahCaminiti1 — APPROVED** [review]
Looking at the whole registration document in another ticket. These look great

**Pull request merged**
